### ISSUE TITLE
`promu info` returns incomplete information when git remote is not set to origin

### ISSUE THREAD

**Issue by kfdm** [prometheus/promu#100]
I encountered this while testing Thanos. For my own testing, I had two remotes. 'upstream' and 'mirror' so when I run `promu info` I'm missing 'Name' and several other fields.

```
$ promu info 
Name: .
Version: v0.1.0-rc.1
Owner: .
Repo: 
Branch: HEAD
Revision: eaf77566ad8a6551e99e1c9beac8cede5e5d61fc
```

This resulted in some confusing bugs with `promu crossbuild tarballs`

While I have not done extensive testing yet, I suspect it may be related to this line

https://github.com/prometheus/promu/blob/b0f9e6cdc0adcd57a447d0cabdcc187dab186afc/cmd/info.go#L84

**Comment by pgier** [comment]
Looks like the error is being swallowed by the shellOutput function.  I'm not sure what's the right fix here, we could just print a warning that origin is required for the relevant info, or we could take the first remote found if there is no remote called 'origin'

**Comment by kfdm** [comment]
That sounds reasonable to me. My checkout not having an `origin` is a bit of an edge case but I think if it printed out a warning, I would have caught that oversight *much* quicker ^^; 
Thank you :)

**Comment by sdurrheimer** [comment]
Closed by #101
